src/main/blackbox/blackbox.c

```diff
@@ -1678,7 +1678,7 @@ STATIC_UNIT_TESTED bool blackboxShouldLogIFrame(void)
 #ifdef USE_GPS
 STATIC_UNIT_TESTED bool blackboxShouldLogGpsHomeFrame(void)
 {
-    if ((GPS_home[0] != gpsHistory.GPS_home[0] || GPS_home[1] != gpsHistory.GPS_home[1]
+    if ((GPS_home_llh.lat != gpsHistory.GPS_home[0] || GPS_home_llh.lon != gpsHistory.GPS_home[1]
```

Comment:
```suggestion
    if ((GPS_home_llh.lat != gpsHistory.GPS_home[GPS_LATITUDE] || GPS_home_llh.lon != gpsHistory.GPS_home[GPS_LONGITUDE]
```

Reply:
But maybe I'll simply store `gpsLocation_t` in `gpsHistory`